t/fcm-loc-layout/00-simple.t

```diff
@@ -20,6 +20,8 @@
 # Basic tests for "fcm status".
 #-------------------------------------------------------------------------------
 . $(dirname $0)/test_header
+
+skip_all "@TODO Skipped to check GH Actions"
```

Comment:
@dpmatthews 
I'll leave this here, and you can chose whether you want to merge with or without the skip.
```suggestion
```